### PR TITLE
Aqua/fix/not copy license tag

### PR DESCRIPTION
**Pull request by mingkang111**

### CODE REVIEW

File: ads/aqua/finetune.py [hunk 1]
```diff
@@ -433,6 +431,9 @@ def create(
             for metadata in ft_model_custom_metadata.to_dict()["data"]
         ]
 
+        if source.freeform_tags:
+            source.freeform_tags.pop(Tags.LICENSE)
```
Comment (lu-ohai) [inline ads/aqua/finetune.py:435]: `source.freeform_tags.pop(Tags.LICENSE.value)` ?

**Review by lu-ohai — CHANGES_REQUESTED** [review]

File: ads/aqua/finetune.py (hunk 1 above)
Comment (mingkang111) [reply ads/aqua/finetune.py]: Good catch! Fixed.

File: ads/aqua/finetune.py [hunk 2]
```diff
@@ -433,6 +431,9 @@ def create(
             for metadata in ft_model_custom_metadata.to_dict()["data"]
         ]
 
+        if source.freeform_tags:
```
Comment (VipulMascarenhas) [inline ads/aqua/finetune.py:434]: can we also add `Tags.LICENSE.value in source.freeform_tags` check here? Service models will have this all the time, but if some users are trying this using CLI, they might try to create a FT model which doesn't have this tag.

Comment (mingkang111) [reply ads/aqua/finetune.py]: I added the check. But we don't need to check if value in tags. `pop` with default value will handle it.

**Review by lu-ohai — APPROVED** [review]

**Review by VipulMascarenhas — APPROVED** [review]

**Pull request merged**